GillianCore/CommandLine/CommandLine.ml

```diff
@@ -447,6 +447,7 @@ struct
         incremental
         entry_point
         () =
+      let () = Fmt_tty.setup_std_outputs () in
```

Comment:
I think this is separate from the spec vars?

Reply:
That is because, plot twist, failure during wpst would not result in any failure at all